packages/next/src/server/app-render/make-get-server-inserted-html.tsx

```diff
@@ -40,7 +40,7 @@ export function makeGetServerInsertedHTML({
       const error = serverCapturedErrors[flushedErrorMetaTagsUntilIndex]
       flushedErrorMetaTagsUntilIndex++
 
-      if (isNotFoundError(error)) {
+      if (isHTTPAccessFallbackError(error)) {
```

Comment:
this one also renders as if is not found below

Reply:
add the metadata handling, they're not taking effects until the new conventions added